contracts/Comptroller.sol

```diff
@@ -503,10 +504,7 @@ contract Comptroller is ComptrollerV4Storage, ComptrollerInterface, ComptrollerE
 
         /* The liquidator may not repay more than what is allowed by the closeFactor */
         uint borrowBalance = CToken(cTokenBorrowed).borrowBalanceStored(borrower);
-        (MathError mathErr, uint maxClose) = mulScalarTruncate(Exp({mantissa: closeFactorMantissa}), borrowBalance);
-        if (mathErr != MathError.NO_ERROR) {
-            return uint(Error.MATH_ERROR);
-        }
+        uint maxClose = mul_ScalarTruncate(Exp({mantissa: closeFactorMantissa}), borrowBalance);
```

Comment:
New naming convention `mul_ScalarTruncate`?

Reply:
Is this bc the other names are conflicting? Would be cool to completely eliminate those old fns and reduce code size

Reply:
Totally with you on the path to eventually clean up Exponential.sol and removing all the alternative functions with underscores (as we’ve been following your helpful post when freeing up space for the patch). We haven’t tackled all the suggested changes - as mentioned in the developer call, we think Quanstamp could take on the broader refactoring needed for Comptroller while our focus has been to get this patch shipped

Reply:
Gotcha. Asked in a separate comment but do we know the code size / gas cost impact of these changes then?